docs/en/5-0-upgrade-guide.rst

```diff
@@ -0,0 +1,215 @@
+5.0 Upgrade Guide
+#################
+
+.. warning::
+    CakePHP ElasticSearch 5.x requires CakePHP 5.0+, ElasticSearch 9.x, and PHP 8.1+.
+
+Requirements
+============
+
+* CakePHP 5.0+
+* ElasticSearch 9.x
+* Elastica 9.x (via ruflin/elastica)
+* PHP 8.1+
```

Comment:
Cake 5.3 is PHP 8.2+. Should this change?

Reply:
I think it's ok as-is. PHP 8.1 won't let them use Cake 5.3 so I think that would prevent any misunderstanding.